torchaudio/functional.py

```diff
@@ -1287,15 +1287,26 @@ def phaser(
     delay_pos = 0
     mod_pos = 0
 
-    output_waveform = torch.zeros_like(waveform, dtype=dtype, device=device)
+    # TODO: Autograd support?
```

Comment:
I don't think the current implementation supports autograd. Among other things, the for loop doesn't connect to the original waveform?

Reply:
What do you mean by original waveform?

Reply:
What I mean is: Can you clarify the comment "TODO: Autograd support?" ?